packages/lodestar/src/node/nodejs.ts

```diff
@@ -54,45 +54,67 @@ export class BeaconNode {
       }
     );
     this.config = config;
-    this.logger = logger;
-
+    this.logger = logger || new WinstonLogger();
     this.reps = new ReputationStore();
     this.db = new BeaconDB({
       config,
-      controller: new LevelDbController(this.conf.db, {
-        logger: this.logger,
-      }),
+      controller: new LevelDbController(
+        {
+          ...this.conf.db,
+          loggingLevel: this.conf.loggingOptions[Module.DATABASE],
+        },
+        {logger}),
```

Comment:
this aint right, because you've passed logger so it will ovveride logging level and module. That said, controller initialization should happen inside BeaconDb and get logger from BeaconDb